templates/register/outside/create_user.html

```diff
@@ -3,47 +3,99 @@
 <html lang="en">
 
 <head>
-	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">
-	<meta charset="utf-8">
-	{% include "header.html" %}{% block head-extra %}{% endblock %}
-	<link href="{% static 'registration_style.css' %}" rel="stylesheet" media="screen">
-	<script src="{% static 'js/setmobile.js' %}" type="text/javascript" charset="utf-8" async defer></script>
+  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />
+  <meta charset="utf-8" />
+  {% include "header.html" %}{% block head-extra %}{% endblock %}
+  <link href="{% static 'registration_style.css' %}" rel="stylesheet" media="screen" />
+  <script src="{% static 'js/setmobile.js' %}" type="text/javascript" charset="utf-8" async defer></script>
+  <script src="{% static 'js/stepform.js' %}" type="text/javascript" charset="utf-8" async defer></script>
 </head>
 
 <body>
-	<div class="container">
-		<div class="row">
-			<div class="col-md-6 col-md-offset-3">
-				<img src="{% static 'images/armada_logo_text_left_black.svg' %}" alt="Armada Ship"
-					class="main-logo-small">
-			</div>
-		</div>
+  <div class="container">
+    <div class="col-md-6 col-md-offset-3">
+      <img src="{% static 'images/armada_logo_text_left_black.svg' %}" alt="Armada Ship" class="main-logo-small" />
+    </div>
 
-		<div class="row header-row">
-			<div class="col-md-12">
-				<h1>Create a new account</h1>
-			</div>
-		</div>
+    <div class="row header-row">
+      <div class="col-md-12 text-center">
+        <p class="h1">Create a new account</p>
+      </div>
+    </div>
 
+    <!--
```

Comment:
Why is this block commented out? Is there another place where you can create a new company? Can we remove the commented out code?